resources/js/features/game-list/components/DataTableToolbar/DataTableDesktopToolbar.tsx

```diff
@@ -38,32 +42,52 @@ export function DataTableDesktopToolbar<TData>({
 
   const { t } = useTranslation();
 
+  const [isCurrentlyPersistingView, setIsCurrentlyPersistingView] = useAtom(
+    isCurrentlyPersistingViewAtom,
+  );
+
   const allColumns = table.getAllColumns();
 
   const currentFilters = table.getState().columnFilters;
   const isFiltered = getAreNonDefaultFiltersSet(currentFilters, defaultColumnFilters);
 
   return (
     <div className="flex w-full flex-col justify-between gap-2">
-      <div className="flex flex-col gap-2 rounded bg-embed p-2 sm:flex-row sm:gap-2 md:gap-3">
-        {doesColumnExist(allColumns, 'system') && filterableSystemOptions?.length > 1 ? (
-          <DataTableSystemFilter table={table} filterableSystemOptions={filterableSystemOptions} />
-        ) : null}
-
-        {doesColumnExist(allColumns, 'achievementsPublished') ? (
-          <DataTableAchievementsPublishedFilter table={table} />
-        ) : null}
-
-        {doesColumnExist(allColumns, 'progress') ? <DataTableProgressFilter table={table} /> : null}
-
-        {isFiltered ? (
-          <DataTableResetFiltersButton
-            table={table}
-            defaultColumnFilters={defaultColumnFilters}
-            tableApiRouteName={tableApiRouteName}
-            tableApiRouteParams={tableApiRouteParams}
+      <div className="flex w-full flex-col items-center justify-between gap-3 rounded bg-embed py-2 pl-2 pr-3 sm:flex-row">
+        <div className="flex w-full flex-col gap-2 sm:w-auto sm:flex-row md:gap-3">
+          {doesColumnExist(allColumns, 'system') && filterableSystemOptions?.length > 1 ? (
+            <DataTableSystemFilter
+              table={table}
+              filterableSystemOptions={filterableSystemOptions}
+            />
+          ) : null}
+
+          {doesColumnExist(allColumns, 'achievementsPublished') ? (
+            <DataTableAchievementsPublishedFilter table={table} />
```

Comment:
At MD or smaller, the filters always show "1", regardless of the value
![image](https://github.com/user-attachments/assets/c836a182-6baf-4f3e-96fd-9099bfaf5863)

Reply:
This also happens in prod (unrelated to this branch). May require some careful thought to solve. 

The filters are showing "1" to represent they currently hold a single value (try filtering by multiple systems and the number will increment on the system filter). I think what may make sense is for the System numeric to continue functioning as-is at these breakpoints, and maybe have a different treatment for the other filters. We're awfully cramped for space, though.

May be best to address this in a different branch.